bridge/tslint.json

```diff
@@ -1,5 +1,10 @@
 {
   "extends": "tslint:recommended",
+  "linterOptions": {
+    "exclude": [
+      "client/app/_models/trace.mock.ts"
```

Comment:
Is here an wildcard like `*.mock.ts` possible?

Reply:
I added it, but only for models. The mocks like DataServiceMock I would not exclude from tslint.